index.html

```diff
@@ -12970,32 +12970,71 @@ <h2>A project for first-time contributions</h2>
 
             <!-- ________ Contributor's card START ________  -->
             <div class="card">
-                <p class="name">Name</p>
+                <p class="name">Subhasmita</p>
                 <p class="contact">
                     <i class="fab fa-twitter"></i>
-                    <a href="YOUR twitter account link" target="_blank">@yourHandle</a>
+                    <a href="https://twitter.com/subh117x" target="_blank">@subh117x</a>
                 </p>
                 <p class="about">
-                    Write here one sentence about yourself. Anything you want: Why do you program, a fun fact, why
-                    do you like programming, what you hope to achieve, your pet's name...
+                    Hello there, i am subhasmita i hope you all are doing well.I am a beginner so would love to help and seek help.I love programming because we could create everything we can possibly think. I hope more people like me would surely contribute to this amazing repository and host your first PR. GoodLuck!
                 </p>
                 <div class="resources">
                     <p>3 Useful Dev Resources</p>
                     <ul>
                         <li>
-                            <a href="#" target="_blank" title="This is a link to an awesome resource">
-                                Link to resources
+                            <a href="https://www.freecodecamp.org/" target="_blank" title="Amazing resources">
+                                Free Code camp
                             </a>
                         </li>
                         <li>
-                            <a href="#" target="_blank"
-                                title="This is another link to web development tool I use a lot">
-                                Link to resources
+                            <a href="www.progate.com" target="_blank"
+                                title="Hands on for beginners and probably the best free resource">
+                                Progate
                             </a>
                         </li>
                         <li>
-                            <a href="#" target="_blank" title="The third link to something useful for developers">
-                                Link to resources
+                            <a href="https://learning.oreilly.com/videos/the-full-stack/9781788470735" target="_blank" title=" O'reilly Web Development Course">
+                                O'reilly Web Development Course
+                            </a>
+                        </li>
+                    </ul>     <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
+       
+                </div>
+            </div>
+            <!-- ________ Contributor's card END ________  -->
+
+
+
+             <!-- ============== TEMPLATE ============== -->
```

Comment:
Just above this line please place the following comments and commit again : ) 

`            <!-- ^^^^ Paste YOUR CARD directly ^^ ABOVE ^^ this line ^^^^ -->
            <!-- ^^^^ make sure there is one line of space between your card start and the card end above it ^^^^ -->
            <!-- ^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^ -->
            <!-- ^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^ -->
`

Reply:
Thanks for pointing out, I've fixed it!